pyproject.toml

```diff
@@ -26,10 +26,7 @@ dependencies = [
   "Sphinx >=5,<9",
   "talonfmt >=1.7.4,<2",
   "tree_sitter_talon >=3!1.0,<3!2",
-  "colorlog >=6.7,<7",
-  "packaging >=23.1,<25",
```

Comment:
What replaces packaging?

Reply:
Hmm I didn't mean to touch those dependencies. That commit was just supposed to organize the ruff list.  I was planning on removing the colorlog and replacing it with rich in another pr, but packaging won't be replaced even then. I'll revert the those lines.